source/chapters/setup.rst

```diff
@@ -4,36 +4,43 @@ Hardware Setup
 .. sectionauthor::
    T.Rafreider <trafreider@mixxx.org>
    S.Brandt <s.brandt@mixxx.org>
+   Be <be.0@gmx.com>
 
 This chapter describes the most common hardware setups to serve as examples. We
-will go over the setup of timecode records/CDs, MIDI control and keyboard
-control.
-
-Audio Output
-============
+will go over the setup of soundcards, controllers, and timecode records/CDs.
 
 Headphone cueing, or just cueing, is previewing the next track you would like to
 mix in your headphones. The audience will not hear what you are cueing in your
-headphones. Being able to cue is a crucial aspect of DJing.
-
-In order to cue with your computer, you will need **at least 2 separate audio
-outputs**. Traditionally, a headphone jack on most laptops **is not a second
-audio output**. Rather, plugging headphones into the jack simply redirects the
-laptop's main output to your headphones. **Having a headphone jack alone will
-not allow you to cue.**
-
-Common Configurations
-=====================
-
-Unlike some commercial DJ systems, Mixxx can control multiple audio devices
-simultaneously.  Depending on your budget and application area your sound setup
-and requirements may vary.  This section provides useful information for club,
-hobby and radio DJs alike.
+headphones. Being able to cue is a crucial aspect of DJing that separates DJing
+from merely making a playlist.
+
+In order to use headphone cueing, you will need at least 2 separate audio
+outputs, typically provided by a USB soundcard (also known as an “audio 
+interface”). The headphone jack on most laptops is not a second audio output. 
+Rather, plugging headphones into the jack simply redirects the laptop's single
+stereo output to your headphones. However, a splitter cable can be used to
+separate the stereo output of a headphone jack into two separate mono outputs
+for headphone cueing.
```

Comment:
"... for headphone cueing, but you will have a much better experience with a dedicated sound card that has at least two stereo outputs.  See [wiki link] for a list of recommended sound cards."